examples/neural_turing_machine_copy.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import numpy as np
+np.random.seed(123)
+import matplotlib.pyplot as plt
+
+from theano import function
+
+from keras.models import Sequential
+from keras.layers.core import TimeDistributedDense, Activation
+from keras.layers.recurrent import LSTM
+from keras.optimizers import Adam
+
+from seya.sandbox.ntm import NeuralTuringMachine as NTM
+
+"""
+Copy Problem defined in Graves et. al [0]
+After 3000 gradients updates, the accuracy becomes >92%.
+
+
+[0]: http://arxiv.org/pdf/1410.5401v2.pdf
+"""
+
+batch_size = 100
+
+h_dim = 128
+n_slots = 121
+m_length = 20
+input_dim = 8
+lr = 1e-3
+clipnorm = 10
+
+##### Neural Turing Machine ######
+
+ntm = NTM(h_dim, n_slots=n_slots, m_length=m_length, shift_range=3,
+          inner_rnn='lstm', return_sequences=True, input_dim=input_dim)
+model = Sequential()
+model.add(ntm)
+model.add(TimeDistributedDense(input_dim))
+model.add(Activation('sigmoid'))
+
+sgd = Adam(lr=lr, clipnorm=clipnorm)
+model.compile(loss='binary_crossentropy', optimizer=sgd)
+
+# LSTM - Run this for comparison
```

Comment:
Right, compiling a model we don't intend on running will probably be a pain for users. Maybe just a flag to route execution to this model, with a comment to explain?

It's a nitpick, I just want to avoid having a section of code commented-out in something that is user-facing.